transforms/this-render-migration/index.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+
+const { getParser } = require('codemod-cli').jscodeshift;
+const { addImportStatement, writeImportStatements } = require('../utils');
+
+module.exports = function transformer(file, api) {
+  const j = getParser(api);
+  const root = j(file.source);
+
+  /**
+   * Replace deprecated this.render() with render() from '@ember/test-helpers' package
+   */
+  function transform() {
+    root.find(j.ExpressionStatement, {
+      expression: {
+        type: 'AwaitExpression',
+        argument: {
+        	type: 'CallExpression',
```

Comment:
Minor point, but please fix the indentation here and on L34. (the repo is quite old, no prettier yet 😔)